data/competencies.yml

```diff
@@ -70,6 +72,10 @@
     - Fixes vulnerabilities in dependencies raised by Snyk
     - Fixes vulnerabilities raised by the cyber security team
     - Doesn't introduce vulnerabilities outlined in the Open Web Application Security Project (OWASP) top 10
+    - Patches EUC endpoints or servers
+    - Maintains anti-virus services 
+    - Reviews Active Directory group policies
+    - Improves the Security Risk Score
```

Comment:
feels like there's a lot of examples here and they're quite specific, can these be distilled down?

Reply:
@apaleslimghost,  Finally getting round to looking at this again,  i think having quite a few  examples is probably a good thing, particularly for something as wide reaching as maintaining security (there are some many attack vectors there are likely to be lots of possible ways to foil the baddies) .. However that said i suggest removing completely the "doesn't introduce vulnerabilities..." example (as it is a kinda anti example) and rewording "Patches EUC endpoints or servers" to "Patches servers or multiple user devices"